op-proposer/project.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@eth-optimism/op-proposer",
+  "$schema": "../node_modules/nx/schemas/project-schema.json",
+  "projectType": "application",
+  "sourceRoot": "op-proposer",
+  "targets": {
+    "build": {
+      "executor": "@nx-go/nx-go:build",
+      "options": {
+        "outputPath": "dist/op-proposer",
+        "main": "op-proposer/cmd/main.go"
+      }
+    },
+    "serve": {
```

Comment:
I'm not sure this makes much sense - there is a lot of config required for these services.